engines/azure/app/models/azure/interface.rb

```diff
@@ -0,0 +1,358 @@
+module Azure
+  require 'base64'
+
+  class Interface
+    class LoginError < StandardError; end
+
+    RESOURCE = 'https://management.azure.com/'
+    HOST = 'https://management.azure.com'
```

Comment:
`RESOURCE` and `HOSTS` are the same, there should be only one

Reply:
Fixed in 9f23e7d . Thanks for catching that.